MapboxNavigation/Resources/Navigation.storyboard

```diff
@@ -101,14 +101,32 @@
-                                <rect key="frame" x="165" y="180" width="44" height="44"/>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Dry-gO-T7u" customClass="MBButton">
+                                <rect key="frame" x="10" y="110" width="40" height="40"/>
+                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="calibratedRGB"/>
+                                <constraints>
+                                    <constraint firstAttribute="width" constant="40" id="Boe-Xa-uFI"/>
+                                    <constraint firstAttribute="height" constant="40" id="ybt-ny-GPN"/>
+                                </constraints>
+                                <inset key="contentEdgeInsets" minX="0.5" minY="0.0" maxX="0.0" maxY="0.0"/>
+                                <state key="normal" image="overview"/>
+                                <connections>
+                                    <action selector="toggleOverview:" destination="3z2-H5-unA" eventType="touchUpInside" id="2Bc-fq-C2C"/>
+                                </connections>
+                            </button>
+                            <button opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="5HY-QD-dBq" customClass="MBHighlightedButton">
+                                <rect key="frame" x="10" y="189.5" width="112" height="42"/>
                                 <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                 <constraints>
-                                    <constraint firstAttribute="width" constant="44" id="Cby-DX-AvF"/>
-                                    <constraint firstAttribute="height" constant="44" id="QGu-S5-V3G"/>
+                                    <constraint firstAttribute="height" constant="42" id="QGu-S5-V3G"/>
+                                    <constraint firstAttribute="width" constant="112" id="auw-yF-4hd"/>
                                 </constraints>
-                                <state key="normal" image="location"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="16"/>
+                                <inset key="contentEdgeInsets" minX="0.0" minY="0.0" maxX="10" maxY="0.0"/>
+                                <inset key="titleEdgeInsets" minX="15" minY="0.0" maxX="0.0" maxY="0.0"/>
+                                <state key="normal" title="Resume" image="location">
```

Comment:
This needs to be localized. In the File inspector, click the Make Localizable button. When we merge this PR, we'll need to add Navigation.strings to Transifex and set up autoupdating.